pkg/backup/backed_up_items_map.go

```diff
@@ -0,0 +1,87 @@
+/*
+Copyright the Velero contributors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package backup
+
+import (
+	"fmt"
+	"sort"
+	"sync"
+)
+
+// backedUpItemsMap keeps track of the items already backed up for the current Velero Backup
+type backedUpItemsMap struct {
+	*sync.RWMutex
+	BackedUpItems map[itemKey]struct{}
```

Comment:
Seems there's no need to expose "BackedUpItems".  i.e. BackedUpItems -> backedUpItems

Reply:
This isn't referenced from outside, so you're right -- no need to expose.